src/mirakuru/http.py

```diff
@@ -59,19 +65,22 @@ def __init__(self, command, url, **kwargs):
         if port is None:
             port = self.DEFAULT_PORT
 
+        self.status = str(status)
+        self.status_re = re.compile(str(status))
+
         super(HTTPExecutor, self).__init__(
             command, host=self.url.hostname, port=port, **kwargs
         )
 
     def after_start_check(self):
-        """Check if defined url returns successful head."""
+        """Check if defined URL returns expected status to a HEAD request."""
         try:
             conn = HTTPConnection(self.host, self.port)
 
             conn.request('HEAD', self.url.path)
-            response = conn.getresponse()
+            status = str(conn.getresponse().status)
```

Comment:
[HTTPResponse.status](https://docs.python.org/3/library/http.client.html) seems to be an integer in both python versions.